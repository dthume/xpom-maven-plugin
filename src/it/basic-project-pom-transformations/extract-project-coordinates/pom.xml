<?xml version="1.0" encoding="UTF-8" ?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation=" http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  
  <modelVersion>4.0.0</modelVersion>

  <parent>
    <groupId>org.dthume.maven.xpom.it</groupId>
    <artifactId>basic-project-pom-transformations</artifactId>
    <version>0.1-SNAPSHOT</version>
    <relativePath>../pom.xml</relativePath>
  </parent>

  <artifactId>extract-project-coordinates</artifactId>
  <packaging>pom</packaging>
  
  <properties>
    <useEffectiveModel>true</useEffectiveModel>
  </properties>
  
  <build>
    <plugins>
      <plugin>
        <groupId>@project.groupId@</groupId>
        <artifactId>@project.artifactId@</artifactId>
        <configuration>
          <useEffectivePOM>true</useEffectivePOM>
        </configuration>
      </plugin>
    </plugins>
  </build>
</project>
