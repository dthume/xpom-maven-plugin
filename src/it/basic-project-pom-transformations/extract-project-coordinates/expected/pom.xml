<?xml version="1.0" encoding="UTF-8"?>
<project>
  <groupId>org.dthume.maven.xpom.it</groupId>
  <artifactId>extract-project-coordinates</artifactId>
  <version>0.1-SNAPSHOT</version>
</project>